kubespawner/spawner.py

```diff
@@ -433,6 +442,15 @@ def _hub_connect_port_default(self):
         """
     )
 
+    user_storage_match_labels = Dict(
+        help="""
```

Comment:
So by default its nothing so the default behavior should be the same.

No, it still creates a PVC spec as it was doing before. This new property basically it just passes the `matchLabels` field from the PVC spec so its possible to filter between existing PV. Lets say you have multiple PV you can select which one you want.

From: https://kubernetes.io/docs/concepts/storage/persistent-volumes/ - PersistentVolumeClaims section.